### ISSUE TITLE
Upload not working on Safari for iOS

### ISSUE THREAD

**Issue by axelburks** [mozilla/send#705]
When I upload a file it would say that "A problem occurred with this web page so it was reloaded".

**Comment by dannycoates** [comment]
Thanks for the report @axelburks and sorry it didn't work. It'll help us track down the problem if you can answer as many of these as possible:

- What device this was? (iPhone 6, iPad Pro, etc)
- What version of iOS?
- How big the file was?
- What type of file? (jpeg, video, text)
- Were you uploading on Wifi or cellular?

**Comment by axelburks** [comment]
- What device this was? (iPhone 6, iPad Pro, etc)
iPhone X and iPhone 7P
- What version of iOS?
iOS 11.2.2 and 11.2.5b6
- How big the file was?
293M
- What type of file? (jpeg, video, text)
mp4
- Were you uploading on Wifi or cellular?
wifi and cellular

**Comment by axelburks** [comment]
@dannycoates 
And download dosen't work too. When download is complete, it shows the file as unknown file which dosen't have filename and extension name.
![img_5622](https://user-images.githubusercontent.com/11376445/35083337-9c1416c2-fc59-11e7-997a-f3c46e25caaa.PNG)
![photo_2018-01-18_14-13-13](https://user-images.githubusercontent.com/11376445/35083358-bd86f9d2-fc59-11e7-8b4b-e78425b5d630.jpg)

Emmm.... And the layout of download page is weird.
![img_5620](https://user-images.githubusercontent.com/11376445/35083333-958977d4-fc59-11e7-8ee3-ae75581a8960.PNG)
![img_5621](https://user-images.githubusercontent.com/11376445/35083335-99db5e60-fc59-11e7-83c2-9a78dd1b6fc7.PNG)

**Comment by wresuolc** [comment]
@SoftVision-PaulOiegas - can softvision try to reproduce this?

**Comment by SoftVision-PaulOiegas** [comment]
Summoning @Softvision-MariusComan with his rose gold iPhone to test this and share the results. 😆

**Comment by Softvision-MariusComan** [comment]
Indeed, the download page UI is not adapting to screen resolution and you need to swipe the page to see it all (no responsive UI). Haven't seen any issues on the upload page however.

Also, I can confirm that the file looses the name after download is finished, but on my end the file type is kept. I don't understand the language in the screenshots to tell for sure if it's the same on my end.
I think that the user might also loose the file type because he tries to download an ".apk" file which is an Android file format and not an iOS file format. maybe the OS does not know how to interpret it and looses the format?
![img_2325](https://user-images.githubusercontent.com/31243259/35278824-d77303c4-0053-11e8-8401-560063040617.PNG)
![img_2326](https://user-images.githubusercontent.com/31243259/35278825-d799b47e-0053-11e8-832a-07ce475017c5.PNG)

**Comment by axelburks** [comment]
@Softvision-MariusComan 
About apk files, I know that iOS can not regonize it. I just want to keep the file's name wih the extension name, e.g., "abc.apk".

And please check the upload issue above.
